js/src/components/modals/SemRushLoading.js

```diff
@@ -0,0 +1,31 @@
+/* External dependencies */
+import React from "react";
+import { __, sprintf } from "@wordpress/i18n";
+import { Fragment } from "@wordpress/element";
+import { SvgIcon } from "@yoast/components";
+
+/**
+ * Creates the loading content for the SEMRush related keywords modal.
+ *
+ * @returns {React.Element} The SEMRush loading content.
+ */
+const SemRushLoading = () => {
+	return (
+		<Fragment>
+			<p>
```

Comment:
For better alignment of the text and the icon we may want to use flexbox and align items center. While CSS stuff should be discussed with the Frontend team, in the meantime I'd add a className to the paragraph following BEM naming conventions.

Reply:
What class name do you suggest?

Reply:
The modal itself has a `yoast-related-keyphrases-modal` class. This is an element within the modal. I'd go with `yoast-related-keyphrases-modal__loading`. Best person to ask is @JoseeWouters